execute/executetest/table.go

```diff
@@ -480,6 +481,88 @@ func NormalizeTables(bs []*Table) {
 	for _, b := range bs {
 		b.Key()
 	}
+	bs = sortByGroupKey(bs)
+}
+
+// sortByGroupKey uses recursive quicksort to sort the tables by group key
+// in alphabetical order
```

Comment:
Implemented two different versions of quick sort for this PR. I used recursive quick sort for table order, since the tables were easy to reorder independently. And I used iterative quick sort for column order, since columns need to be updated per table, it was easier to keep track of them using a stack.